tests/libnet/ping.go

```diff
@@ -47,7 +48,7 @@ func PingFromVMConsole(vmi *v1.VirtualMachineInstance, ipAddr string, args ...st
 	args = append([]string{pingString, ipAddr}, args...)
 	cmdCheck := strings.Join(args, " ")
 
-	err := VmiConsoleRunCommand(vmi, cmdCheck, maxCommandTimeout)
+	err := console.VmiConsoleRunCommand(vmi, cmdCheck, maxCommandTimeout)
```

Comment:
If we are ok changing names at this PR what about `console.RunCommandOnVMI(vmi...`

Reply:
If we name the package `vmiconsole` It would read like `vmiconsole.RunCommand` does not look bad.

Reply:
Does not worth it and `console` concept is quite clear on these tests.